rio_tiler/io/xarray.py

```diff
@@ -239,6 +239,11 @@ def tile(
         arr = ds.to_masked_array()
         arr.mask |= arr.data == ds.rio.nodata
 
+        output_bounds = ds.rio._unordered_bounds()
+        if output_bounds[1] > output_bounds[3] and ds.rio.transform().e > 0:
+            yaxis = self.input.dims.index(self.input.rio.y_dim)
+            arr = numpy.flip(arr, axis=yaxis)
```

Comment:
Since we do the same operation three times in the same file it might be useful to add a utility function that we could recycle.

Reply:
we could, maybe later :-)